src/kodeagent/kodeagent.py

```
@@ -1288,8 +1288,10 @@ async def _act(self) -> AsyncIterator[AgentResponse]:
 
             except Exception as ex:
                 error_msg = f'*** Error running code: {type(ex).__name__}: {str(ex)}'
-                logger.exception(error_msg)
-                self.add_to_history(ChatMessage(role='user', content=error_msg))
+                logger.error(error_msg)
```

Ruff (0.14.6)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)